src/resolvers/Mutation/removeAdvertisement.ts

```diff
@@ -21,7 +21,7 @@ export const removeAdvertisement: MutationResolvers["removeAdvertisement"] =
 
```

Comment:
`_id: args.id ? args.id : ""` -> `_id: args.id`

rename `currentAd` to `advertisement` or `existingAdvertisement`

also why is there no check here for who is allowed to delete the advertisement?

Reply:
Ads management is done by ADMIN and SUPER ADMIN and USER can not access the admin portal.
So the check for who is allowed to delete an advertisement shouldn't be required.

What do you think?

Also the `removeAdvertisement` Mutation is not being used anywhere instead `src/resolvers/Mutation/deleteAdvertisementById.ts` is used. 

Can we update the `deleteAdvertisementById` mutation and remove the `removeAdvertisement` Mutation?

Reply:
it's not about the admin portal, it's about general security, you're assuming that no user would ever make a `removeAdvertisement` graphql mutation request through a portal other than admin portal, that's a naive assumption

yeah sure, get rid of one of them, also if `deleteAdvertisementById` is kept, rename it to `deleteAdvertisement`